### DISCUSSION TITLE
Is there no settings for auth providers in self hosted dashboard?

### DISCUSSION THREAD

**Discussion by adityadhawan22** [supabase/supabase#8566] [Questions] [answered]
I can see the settings for auth providers in managed supabase:
![image](https://user-images.githubusercontent.com/15171755/185792598-a0a83ffe-d951-483a-bbbc-22558d743d43.png)

No options in self hosted: 
<img width="281" alt="Screenshot 2022-08-21 at 14 15 39" src="https://user-images.githubusercontent.com/15171755/185792652-8a3a6d2c-af2a-4408-91b7-e4c46796bc44.png">

**Comment by GaryAustin1** [comment, marked as answer]
No, those are set using a config file in local development.

https://supabase.com/docs/guides/hosting/overview#configuration

https://github.com/supabase/gotrue/blob/master/example.env

https://github.com/supabase/gotrue#external-authentication-providers
